packages/lib/src/globalConfig/globalConfig.ts

```diff
@@ -1,3 +1,4 @@
+import { v4 as uuidv4 } from "uuid"
```

Comment:
it's better to use 'uuid/v4` or you have a proper setup of code-splitting?

Reply:
nice catch, will change it on another pr